### PR TITLE
Appscale services start/stop commands (serving status modification)

### PR DESCRIPTION
**Pull request by sjones4**
Adds start and stop command for application/project/service (default version):

```
appscale services start <project_id> <service_id>
appscale services stop <project_id> <service_id>
```

This modifies the serving status for the application version.

Relates to Appscale/appscale#2894

### CODE REVIEW

File: appscale/tools/admin_api/client.py [hunk 1]
```diff
@@ -147,6 +147,45 @@ def delete_version(self, project_id, service_id, version_id):
...
+                    .format(prefix=self.prefix, project=version.project_id,
+                            service=version.service_id, version=version.id))
+    headers = {
+      'AppScale-Secret': self.secret,
+      'Content-Type': 'application/json'
+    }
+    parameters = {
+      'updateMask': ','.join(fields)
+    }
+    query_parameters = '&'.join('='.join(item) for item in parameters.items())
```
Comment (cdonati) [inline appscale/tools/admin_api/client.py]: I would recommend using `urllib.urlencode(parameters)` for this use case. An alternative would be using the `params` argument to `requests.patch`.

File: appscale/tools/appscale_tools.py [hunk 2]
```diff
@@ -630,6 +631,65 @@ def remove_app(cls, options):
...
+        passed in via the command-line interface.
+    Raises:
+      AppScaleException: If the named service isn't running in this
+        AppScale cloud, or if start is not valid for the service.
+    """
+    login_host = LocalState.get_login_host(options.keyname)
+    secret = LocalState.get_secret_key(options.keyname)
+    admin_client = AdminClient(login_host, secret)
+
+    version = Version(None, None)
```
Comment (cdonati) [inline appscale/tools/appscale_tools.py]: Should this just be `Version(None)`? It looks like it just takes the runtime parameter.

Comment (sjones4) [reply appscale/tools/appscale_tools.py]: I think `Version(None, None)` is correct for `master`:

```
class Version(object):
  """ Represents an Admin API Version resource. """
  def __init__(self, runtime, config_type):
```

https://github.com/AppScale/appscale-tools/blob/master/appscale/tools/admin_api/version.py#L22

Comment (cdonati) [reply appscale/tools/appscale_tools.py]: You're right. I was looking at an older commit.

File: appscale/tools/admin_api/client.py (hunk 1 above)
Comment (sjones4) [reply appscale/tools/admin_api/client.py]: Updated to pass parameters directly to `requests.patch`. Thanks for the suggestion.

**Pull request merged**